src/OrleansVSTools/OrleansVSTools/packages.config

```diff
@@ -2,40 +2,78 @@
 <packages>
   <package id="FSharp.Core" version="4.1.0.2" targetFramework="net451" />
```

Comment:
We've update F# packages to newer versions, but not sure if we need F# package here?

F# version numbers: https://github.com/dotnet/orleans/blob/master/test/TestFSharp/packages.config